lib/active_merchant/billing/gateways/wompi.rb

```diff
@@ -65,11 +65,12 @@ def refund(money, authorization, options = {})
         # commit('refund', post, '/refunds_sync')
 
         # All refunds will instead be voided. This is temporary.
-        void(authorization)
+        void(authorization, options, money)
       end
 
-      def void(authorization, options = {})
-        commit('void', {}, "/transactions/#{authorization}/void_sync")
+      def void(authorization, options = {}, money = nil)
+        post = money ? { amount_in_cents: amount(money).to_i } : {}
```

Comment:
This logic is an attempt to preserve the functionality of an actual request to the void endpoint (rather than a refund redirected to a void)

I did some local testing (including integration tests) and realized that a /void request would break without this default `nil` amount for `money`, because usually no amount argument is expected for a true void